### ISSUE TITLE
Watch list

### ISSUE THREAD

**Issue by mtekman** [mtekman/org-tanglesync.el#4]
For every dedicated conf file, or conf dir, generate a list of all tangled files and store them to a global var.

Add a save or onload hook so that every time one of those files is opened and modified, the user is prompted to sync the changes back to the source.

Requires:
* map of all tangled files in an org buffer
* ability to resolve duplicates; i.e. tangled files that exist in more than one org buffer.

**Comment by mtekman** [comment]
Have a conf-mode which can be enabled globally to look for syncable buffers, and a set of conf-files to watch

```elisp
(use-package org-tanglesync
    :custom 
    (conf-mode 1)
    :init 
    (setq conf-files '("/path/to/1" "/path/to/2")
```

**Comment by mtekman** [comment]
fixed with v1.1 release
